plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoPageSource.java

```diff
@@ -144,6 +146,7 @@ public Page getNextPage()
                 break;
             }
             currentDoc = cursor.next();
+            totalBytes += currentDoc.toJson().getBytes(StandardCharsets.UTF_8).length;
```

Comment:
Static import `UTF_8`.

Reply:
Calling `toJson` method increases memory usage?

Reply:
Does this JSON represents the actual bytes read from the MongoDB or a different interpretation of the data read from MongoDB ?

Reply:
> Does calling `toJson` method increase CPU & memory usage?

Overall, while there is some impact on CPU and memory usage when invoking the toJSON() method, it is generally considered to be minor and well-optimized. The impact becomes more noticeable for larger or more complex documents, but it is still within acceptable limits for most use cases. In most cases, it should be unnoticeable.

1. CPU Usage: The CPU impact of the toJSON() method is typically minimal for small and simple documents. While this process incurs some CPU overhead, it is generally efficient and designed to be performant. 

2. Memory Usage: The memory impact of the toJSON() method is also generally minor. The memory required to store the resulting JSON string is temporary and limited to the scope of the method invocation.

Reply:
Does `toJson` method captures the approximate size of the input data or does it include the Json parts like `{` as well ? This `toJson` will be invoked for each row for a pretty big dataset won't it be an overhead ?